db/db.go

```diff
@@ -0,0 +1,227 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"time"
+
+	_ "modernc.org/sqlite"
+)
+
+var DB *sql.DB
+
+// Инициализируем базу данных
+func InitDB() {
+	// Получение пути к файлу базы данных из переменной окружения
+	dbFile := os.Getenv("TODO_DBFILE")
+	if dbFile == "" {
+		// Использование текущей рабочей директории
+		appPath, err := os.Executable()
+		if err != nil {
+			log.Fatal(err)
+		}
+		dbFile = filepath.Join(filepath.Dir(appPath), "scheduler.db")
```

Comment:
Мне не нравиться этот способ, так как если запускать локально `go run main.go`, то будет сделан временный файл и запущен где-то и БД будет где-то далеко (в моем случае это был ` /tmp/go-build2315668530/b001/exe/scheduler.db`)
А если мы хотим сделать по месту вызова команды, то нужно сделать просто
```suggestion
		dbFile ="scheduler.db"
```

Reply:
fxd, спасибо!